Website/Website/auth.js

```diff
@@ -89,6 +89,10 @@ function initialize(host) {
     const authContext = new AuthenticationContext(`https://login.microsoftonline.com/${tenantName}`);
     const serviceClusterUrl = env.serviceClusterUrl;
 
+    // Updates will be made once the ARM support is added.
+    // This expects a secret with a value that is a json object comprising of all the backend service names and the external IP addresses as exposed by AKS where each of them can be listened to.
+    const kubernetesServices = JSON.parse(env.kubernetesServices);
```

Comment:
Can you show an example of the value of env.kubernetesServices?

Reply:
what if env.kubernetesServices is null?

Reply:
Updated the code to do the json parsing of the secret in securedSettings.js